internals/secrethub/acl_check.go

```diff
@@ -43,8 +45,21 @@ func (cmd *ACLCheckCommand) Run() error {
 		return err
 	}
 
-	levels, err := client.AccessRules().ListLevels(cmd.path.Value())
-	if err != nil {
+	path := cmd.path.Value()
+
+	levels, err := client.AccessRules().ListLevels(path)
+	if api.IsErrNotFound(err) {
+		isSecret, err := client.Secrets().Exists(path)
```

Comment:
> Question, why would we do an .Exists call here and not simply call ListLevels(secretpath.Parent(path)) immediately?

Because then we would also return the access levels of the parent directory if the child does not exists.

Let's take this tree as an example:
```
workspace/repo/
  dir/
    dir2/
    secret
```

We want `workspace/repo/dir/secret` to return the access levels effective on `workspace/repo/dir`, but we don't want `workspace/repo/dir/foo` to return the access levels effective on `workspace/repo/dir`.